docs/deploy/setup-storageclass.md

```diff
@@ -32,3 +32,5 @@ parameters:
   thinProvision: "true"
 volumeBindingMode: Immediate
 ```
+
+> **_NOTE_: The `reclaimPolicy` in the example is `Retain` which means when you delete a `PVC`, the `PV` under `PVC` will retain, also when you delete the `PV` in `Released` status, the real volume under IOMesh will retain and never removed by Garbage Collection. You can edit the `PV`, change the value of `persistentVolumeReclaimPolicy` to `Delete`. This will remove the real volume under IOMesh. Or you can create a new `StorageClass` with the `reclaimPolicy` change to `Delete`.
```

Comment:
我先按照自己的理解修订一下，仅供参考，然后可以继续讨论修订：

以下是我的理解：这里在说几个属性的取值对操作结果的影响：
（1）一个属性是`reclaimPolicy`，如果取值为`Retain`（保留），则在删除一个PVC时，其中的一个PV（或者多个PVs）将被保留，而当删除一个处在`Released`（释放）状态的PV时，在IOMesh中（和该PV）相应的真实物理卷将会被保留，永远不会被垃圾回收机制删除。另外一种情况与上述相反，可以创建一个新的`StorageClass`，将`reclaimPolicy`的取值设为 `Delete`（删除），这样的话，上述删除操作就可以删除真实物理卷了。感觉将最后一句与第一句合并在一起，统一介绍`reclaimPolicy`的两种可能取值对删除物理卷的影响，会更好些。
（2）另一个属性是`persistentVolumeReclaimPolicy`，可以编辑一个PV，将`persistentVolumeReclaimPolicy`的取值改为`Delete`（删除），这样的话，在删除一个PV（或多个PVs）时，在IOMesh中的真实物理卷将会被删除。

The value of `reclaimPolicy` in the example is `Retain`, which means that, when you delete a `PVC`, the `PV` under the `PVC` will be retained, and when you delete a `PV` in `Released` status, the corresponding real volume in IOMesh will be retained and never be removed by garbage collection (若不是特指，每个单词不用大写开头). On the other hand,  you can create a new `StorageClass` with the value of `reclaimPolicy` set to `Delete`, or edit a `PV` and change the value of `persistentVolumeReclaimPolicy` to `Delete`, so that in both cases, the real volume in IOMesh will be removed.（后半句是新加的，看看是否符合原来写作意图）. 

一些具体改动如下：
The `reclaimPolicy` -> The value of `reclaimPolicy`，添加 value of
is `Retain` which means -> is `Retain`, which means that,，在 which 之前添加一个逗号，在 means 之后添加 that 和一个逗号，这样更好表现下文的两个 when 从句是平行关系
the `PV` under `PVC` will retain -> the `PV` under the `PVC` will be retained，不确定在一个 PVC 之下（或之内），是否可能有多个 PV，如果有的话，用复数 PVs 比较好些，另外再次提到 PVC时，添加定冠词 the
when you delete the `PV` in `Released` status -> when you delete a `PV` in `Released` status，感觉 the 改为 a 更好些
the real volume under IOMesh will retain -> the real volume under IOMesh will be retained , retained 改为 be retained，另外 under 是否换成 in 更好些？还有这里说的 the real volume 是否要和前文的在 `Released` 状态的 `PV` 有关联？如果有的话，明确表达出来比较好些，比如至少加上一个corresponding 
never removed -> never be removed
Garbage Collection -> garbage collection，(若不是特指，每个单词不用大写开头
You can edit the `PV` -> You can edit a `PV` ，这里的PV是否为特指，如果不是的话，前边用不定冠词a比较好
to `Delete`. This will remove -> to `Delete`, and this will remove，改为一个句子
under IOMesh -> in IOMesh
with the `reclaimPolicy` change to `Delete` -> with the value of  `reclaimPolicy` set to `Delete`